packages/internal-test-helpers/lib/test-cases/application.js

```diff
@@ -45,6 +45,10 @@ export default class ApplicationTestCase extends TestResolverApplicationTestCase
     }
   }
 
+  getController(name) {
```

Comment:
I was avoiding using the `QueryParam` test case, because it has a lot of additional API space that would go unused. Adding this to the `moduleFor` sounds like a better plan to me.